From ff67cdd8ef68ba293719bc249555268bc598549c Mon Sep 17 00:00:00 2001
From: Dylan Van Assche <me@dylanvanassche.be>
Date: Sat, 24 Apr 2021 07:44:21 +0200
Subject: [PATCH] audacity.desktop: force X11 instead of Wayland

Audacity  leaks a lot of memory under Wayland when recording.
See https://gitlab.alpinelinux.org/alpine/aports/-/issues/12634
---
 src/audacity.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/audacity.desktop.in b/src/audacity.desktop.in
index 71d3cb4e7..00b210062 100644
--- a/src/audacity.desktop.in
+++ b/src/audacity.desktop.in
@@ -50,7 +50,7 @@ Icon=@AUDACITY_NAME@
 Type=Application
 Categories=AudioVideo;Audio;AudioVideoEditing;
 
-Exec=env UBUNTU_MENUPROXY=0 @AUDACITY_NAME@ %F
+Exec=env UBUNTU_MENUPROXY=0 env GDK_BACKEND=x11 @AUDACITY_NAME@ %F
 StartupNotify=false
 Terminal=false
 MimeType=application/x-audacity-project;@MIMETYPES@
-- 
2.31.1

